### PR TITLE
Use RTable in r_core_debug_rr

### PR DESCRIPTION
**Pull request by x0urc3**
**Your checklist for this pull request**
- [X] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [X] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Detailed description**

Use RTable in r_core_debug_rr. This applies to drr/arr and drrj/arrj

**Test plan**

Eyeball result between PR and master

### CODE REVIEW

**Comment by x0urc3** [comment]
r_core_anal_hasrefs color messed up the column width. Should I just disable color output?

drr

File: libr/core/cmd_debug.c [hunk 1]
```diff
@@ -2023,111 +2023,79 @@ R_API void r_core_debug_ri(RCore *core, RReg *reg, int mode) {
...
-			free (rrstr);
-		} else {
-			if (mode == 'j') {
-				pj_ks (pj, "ref", "");
-				pj_end (pj);
-			} else {
-				r_cons_printf ("\n");
-			}
+
+		RStrBuf *sbname = r_strbuf_new (NULL);
```
Comment (pelijah) [inline libr/core/cmd_debug.c]: r_str_newf ("%s%s%s", color, r->name, colorend)?

File: libr/core/cmd_debug.c [hunk 2]
```diff
@@ -2023,111 +2023,79 @@ R_API void r_core_debug_ri(RCore *core, RReg *reg, int mode) {
...
 	}
+
 	if (mode == 'j') {
-		pj_end (pj);
-		r_cons_printf ("%s\n", pj_string (pj));
-		pj_free (pj);
-		if (use_colors) {
-			r_config_set_i (core->config, "scr.color", use_colors);
-		}
+		r_cons_printf ("%s", r_table_tojson (t));
```
Comment (pelijah) [inline libr/core/cmd_debug.c]: r_cons_print (r_table_tojson (t))?

File: libr/core/cmd_debug.c (hunk 1 above)
Comment (x0urc3) [reply libr/core/cmd_debug.c]: Thanks for the suggestion. Will do this in next PR

File: libr/core/cmd_debug.c (hunk 2 above)
Comment (x0urc3) [reply libr/core/cmd_debug.c]: Will do this in next PR

**Pull request merged**